### PR TITLE
Added Agoric chain + upgrade impacthub chain

### PR DESCRIPTION
**Pull request by betterclever**

### CODE REVIEW

File: impacthub/build.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@ services:
     env_file:
       - ../.env
     volumes:
-      - ./node-data:/root/.ixod
+      - ./node-data:/root/.ixo
```
Comment (tombeynon) [inline impacthub/build.yml:24]: Is `.ixo` the correct directory? We'll need to update the PROJECT_DIR arg if so

File: README.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ tagged with the form `$COSMOS_OMNIBUS_VERSION-$PROJECT-$PROJECT_VERSION`.
 |Project|Version|Image| |
 |---|---|---|---|
 |[akash](https://github.com/ovrclk/akash)|`v0.14.1`|`ghcr.io/ovrclk/cosmos-omnibus:v0.0.27-akash-v0.14.1`|[Example](./akash)|
+|[agoric](https://github.com/Agoric/ag0)|`agoric-3.1`|`ghcr.io/ovrclk/cosmos-omnibus:v0.0.27-agoric-3.1`|[Example](./agoric)|
```
Comment (tombeynon) [inline README.md:30]: Image should be `-agoric-agoric-3.1`

File: Dockerfile [hunk 3]
```diff
@@ -58,9 +61,6 @@ RUN mv $BUILD_PATH/$PROJECT_BIN /bin/$PROJECT_BIN
 #
 FROM debian:buster AS default
 
-ARG PROJECT=akash
```
Comment (tombeynon) [inline Dockerfile:61]: I'm not sure we can remove these ARGs from this build step - what was the reason for these Dockerfile changes?

**Review by tombeynon — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (betterclever) [reply Dockerfile:61]: Due to buildkit not being enabled for my builds earlier, I juggled around some arguments as they were not being passed to steps that needed them. 
Forgot to fix the order post enabling it.

File: impacthub/build.yml (hunk 1 above)
Comment (betterclever) [reply impacthub/build.yml]: ixod is the correct directory. It was .ixo in the docker-compose.yml mount, I thought that's the correct source of truth. I confirmed it by running the node locally now.

**Review by tombeynon — APPROVED** [review]
Looks great! Will release a new version later today

**Pull request merged**